Calibration/IsolatedParticles/test/HLT_IsoTrig_MC_New2.py

```diff
@@ -75,9 +77,8 @@
 
 process.analyze = cms.EndPath(process.IsoTrigHB + process.IsoTrigHE + process.IsoTrackCalibration)
 
-from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, 
-                              'auto:run2_mc_GRun', '')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']

```

Comment:
At line 27, change it to: `process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')`